Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/AutoFramework.java

```diff
@@ -0,0 +1,99 @@
+package org.firstinspires.ftc.team6220_PowerPlay;
+
+
+abstract public class AutoFramework extends BaseAutonomous {
+
+    /**
+     * @param AutoSelector Assign to 0 or 1, corresponding to BlueLeft/RedLeft, or BlueRight/RedRight respectively
+     * @throws InterruptedException If you haven't added a config for that autonomous
+     */
+    public void runAuto(int AutoSelector) throws InterruptedException {
```

Comment:
Please make AutoSelector an enum. It'll still be 0 or 1 underneath, but it'll be easier to understand.